phase1/openstack/README.md

```diff
@@ -11,16 +11,16 @@ $ git clone https://github.com/kubernetes/kubernetes-anywhere.git
 $ cd kubernetes-anywhere
 ```
 
-Hop into deployment shell which includes all the tools you need to deploy a kubernetes-anywhere cluster. Run:
+Create an SSH key pair in the kubernetes-anywhere directory. SSH key pair is needed as a config option during subsequent steps so set to /opt/kubernetes-anywhere/id_rsa when asked:
```

Comment:
`/opt/kubernetes-anywhere` is the directory within the container environment. According to the changes proposed, `ssh-keygen` is called before entering container environment. So i think it should be the current working directory (kubernetes-anywhere directory). but while configuring ssh_key_file during `make config`, it should be an absolute path set to `/opt/kubernetes-anywhere/id_rsa`

Reply:
Apologies, I meant to add also that the keypair is not accessible once you leave the container environment; this is another reason I suggest creating it outside of the container environment via changing the order of steps.

Reply:
oh, i got the meaning now. i got confused because of the phrase `when asked`. is it `ssh-keygen` which will ask for path or the `make config` ?. i think its better to clear it out.

Also Thanks for doing this. This is definitely an improvement in usability.

Reply:
Glad to help! So I just used the previous statement in part, 'If you don't already have an SSH key pair, you can generate one like below. SSH key pair is needed as a config during subsequent steps'

It can be more clear definitely, I understand how it can be confusing.